sdk/eventgrid/azure-eventgrid/samples/eventgrid_basic_samples/async_samples/sample_publish_cloud_event_using_dict_async.py

```diff
@@ -23,7 +23,6 @@
 
 
 async def publish():
-    # To Event Grid Basic
```

Comment:
maybe rename folders to basic and namespace? since eventgrid and samples are implied based on folder structure